src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java

```diff
@@ -186,6 +186,13 @@ private Client setCreateAccountClientDetails(CreateAccountRequest createAccountR
         roleFile.setRegisteredClientRoleCd("FILE");
         roles.add(roleFile);
 
+        if (identityProvider.equalsIgnoreCase("BCSC")) {
```

Comment:
Magic string. Possibly extract to constant.

Reply:
Yes, I will refactor the method and all the constants that should be there.